libs/pika/async_mpi/tests/unit/CMakeLists.txt

```diff
@@ -4,19 +4,20 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-set(tests algorithm_transform_mpi mpi_ring_async_sender_receiver
-          mpi_async_storage
+set(tests # algorithm_transform_mpi
+    mpi_ring_async_sender_receiver
+    # mpi_async_storage
 )
 
 # cmake-format: off
 set(mpi_ring_async_sender_receiver_PARAMETERS
-    ARGS "--in-flight-limit=32"
+    ARGS "--in-flight-limit=32" "--rounds=10" "--iterations=100" "--pika:ignore-process-mask"
     THREADS 4 LOCALITIES 2 RUNWRAPPER mpi
 )
 
 set(mpi_async_storage_PARAMETERS
     ARGS
-        "--in-flight-limit=256" "--localMB=256" "--transferKB=1024" "--seconds=1"
+        "--in-flight-limit=256" "--localMB=256" "--transferKB=1024" "--seconds=1" "-Ipika.stacks.use_guard_pages=0" "--pika:ignore-process-mask"
```

Comment:
Are the added flags just for testing?

Reply:
`rounds` and `iterations` are part of the test, stack guard pages help prevent segfaults and the ignore process mask is because the default `mpiexec` causes warnings since threads are not setup unless we add a bunch of other openmpi/other flags on machines like my laptop

Reply:
`rounds` and `iterations` are fine.

`pika.stacks.use_guard_pages`: do you know why this particular test needs this? Is this something we should consider disabling by default anyway? Or enabling only in debug mode or something like that?

`--pika:ignore-process-mask`: it'd be nice if we could control this with a CMake variable instead (either a flag, or an "extra flags" option).

Reply:
- https://github.com/pika-org/pika/issues/613
- https://github.com/pika-org/pika/issues/614

Reply:
Guard pages are now disabled by default so you can remove that option.